content/en/about.html

```diff
@@ -76,9 +76,12 @@ <h2>Policies</h2>
   <div class="container">
     <h2>Open Source Repositories</h2>
     <p>Most of the code we use to operate Divvi Up is open source.</p>
+    <p>Divvi Up is based on open standards and most of the code we use to operate it is open source.</p>
     <ul>
-      <li><a href="https://github.com/divviup/prio-server">Prio Server on Github</a></li>
-      <li><a href="https://github.com/divviup/libprio-rs">Prio Rust Implementation on Github</a></li>
+      <li>Divvi Up implements the Distributed Aggregation Protocol (DAP) (<a href="https://github.com/ietf-wg-ppm/ppm-specification">specification on Github</a>)</li>
+      <li>DAP is built on Verifiable Distributed Aggregation Functions (VDAF), (<a href="https://github.com/cfrg/draft-irtf-cfrg-vdaf">specification on Github</a>)</li>
+      <li><a href="https://github.com/divviup/libprio-rs">VDAF Rust Implementation on Github</a></li>
```

Comment:
```suggestion
      <li><a href="https://github.com/divviup/libprio-rs">VDAF Implementation in Rust on Github</a></li>
```

(minor phrasing nit; alternatively, perhaps "Rust VDAF Implementation on Github"?)